src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.html

```diff
@@ -51,18 +51,25 @@
     >
     <div class="news-text-container">
       <div class="news-links-images">
-        <img [src]="images.twitter"
-             alt="twitter"
-             class="news-links-img"
-        >
-        <img [src]="images.linkedIn"
-             alt="linkedin"
-             class="news-links-img"
-        >
-        <img [src]="images.facebook"
-             alt="facebook"
-             class="news-links-img last-img"
-        >
+        <a [href]="'https://twitter.com/share?url=' + currentPage + '&text='+newsItem.title+'&hashtags='+newsItem.tags.join(',')" target="_blank">
```

Comment:
maybe, is it better to use a handler for this action?

Reply:
This is my mistake
i am fixing it in another PR